### PR TITLE
update driver installer

### PR DESCRIPTION
**Pull request by Dragoncell**
What type of PR is this?
To fix test failures on GPU installation in https://github.com/kubernetes/kubernetes/issues/124950.

/kind bug
/kind failing-test

What this PR does / why we need it:
Fix GPU test failures in master.

Next step: add tests and get them running again once this fix PRs are merged

Which issue(s) this PR fixes:
Fixes # https://github.com/kubernetes/kubernetes/issues/124950

1.30 pull request: https://github.com/kubernetes/kubernetes/pull/125208

### CODE REVIEW

File: test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml [hunk 1]
```diff
@@ -47,17 +47,43 @@ spec:
...
         hostPath:
           path: /
+      - name: cos-tools
+        hostPath:
+          path: /var/lib/cos-tools
+      - name: nvidia-config
+        hostPath:
+          path: /etc/nvidia
       initContainers:
+      - image: "ubuntu"
```
Comment (BenTheElder) [inline test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml:57]: TODO: pin an image instead of depending on a :latest dockerhub image.

**Comment by BenTheElder** [comment]
> Fixes # https://github.com/kubernetes/kubernetes/issues/124950

We should also update this part of the PR descriptions, since they don't fully fix the issue (we'll still have to add tests and get them running again once your driver install fix PRs are in).

File: test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml (hunk 1 above)
Comment (Dragoncell) [reply test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml]: Updated with specific ubuntu with sha256

**Comment by Dragoncell** [comment]
> > Fixes # #124950
> 
> We should also update this part of the PR descriptions, since they don't fully fix the issue (we'll still have to add tests and get them running again once your driver install fix PRs are in).

Updated the description. Thanks

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**